dev/com.ibm.ws.security.client_fat/build.gradle

```diff
@@ -0,0 +1,1033 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+// TODO earDD.ear, TechnologySamples.ear
+
+dependencies {
+  requiredLibs 'org.apache.httpcomponents:httpclient:4.1.2',
+               'org.apache.httpcomponents:httpcore:4.1.2',
+               'commons-codec:commons-codec:1.4',
+               'commons-logging:commons-logging:1.1.1',
+               project(':com.ibm.websphere.javaee.servlet.3.0'),
+               project(':com.ibm.ws.security.audit_fat.common.tooling')
+}
+
+addRequiredLibraries.dependsOn addJakartaTransformer
+
+/* 
+ * This is where all test applications will be built.
+ */
+def appBuildDir = "${buildDir}/test-application"
+
+
+/******************************************************************
```

Comment:
Shouldn't some of these applications be built automatically with the tooling without having to write them all out in the gradle file?

Reply:
Is there an easy way to do this? If so, I am more than happy to make the changes.

Reply:
I thought you wanted to do it proactively during build and bloat the autoFVT.zip file instead of during execution of the tests.  I don't know if you can do that without what you are doing.  @aguibert would probably know more.

Reply:
I left them in the build.gradle, as we do with all our other projects.